ocfweb/reservations/reserve.py

```diff
@@ -0,0 +1,214 @@
+import email.mime.text
+import subprocess
+from collections import namedtuple
+from email.utils import parseaddr
+
+import ocflib.misc.validators
+from django import forms
+from django.shortcuts import render
+from jinja2 import Environment
+from jinja2 import PackageLoader
+from ocflib.account.search import user_attrs_ucb
+
+from ocfweb.auth import calnet_required
+from ocfweb.component.forms import wrap_validator
+
+SENDMAIL_PATH = '/usr/sbin/sendmail'
+MAIL_FROM = 'Open Computing Facility <help@ocf.berkeley.edu>'
+
+JINJA_MAIL_ENV = Environment(loader=PackageLoader('ocfweb', ''))
+
+
+class NewReservationRequest(namedtuple(
+    'NewReservationRequest', [
+        'real_name',
+        'contact_email',
+        'student_group',
+        'reason',
+        'date',
+        'starttime',
+        'endtime',
+        'handle_warnings',
+    ],
+)):
+    """Request for account creation.
+    :param real_name:
+    :param contact_email:
+    :param student_group:
+    :param reason:
+    :param date:
+    :param starttime:
+    :param endtime:
+    :param handle_warnings: one of WARNINGS_WARN, WARNINGS_SUBMIT
+        WARNINGS_WARN: don't create request, return warnings
+        WARNINGS_SUBMIT: submit for staff approval
```

Comment:
where are you actually using this `handle_warnings` stuff?

Reply:
Nowhere, and removed. Thanks for pointing this out.